woocommerce/woocommerce
# Add contextual product more menu

### All Submissions:






### Changes proposed in this Pull Request:

Adds the product form's more menu and several links.

Note that the CES modal still needs to be completed before it can be hooked up to this component.

<img width="234" alt="Screen Shot 2022-11-01 at 10 42 09 AM" src="https://user-images.githubusercontent.com/10561050/199301444-181f868e-63c5-430e-bc23-09733e0fe734.png">



Closes #35177  .






### How to test the changes in this Pull Request:



1. Navigate to the new product management experience ( Add new (MVP) )
2. Click the "more" (vertical ellipsis) button in the top right of the header
3. Note that each of the links closes the menu
4. Note that clicking "Use the classic editor" takes you to the classic editing experience
5. Navigate back to the new experience and publish a product
6. Click again on "Use the classic editor"
7. Note that you are taken to the classic editor for the respective published product



### Other information:




### FOR PR REVIEWER ONLY:
